### PR TITLE
added changes to check_performance procedure, added a check which mak…

### PR DESCRIPTION
**Pull request by verkerk20**
…e sure the start time of a performance is between the start and end date of a festival

### CODE REVIEW

**Comment by verkerk20** [comment]
[Constraint: De performance start_date moet binnen de festivals start en eind datum vallen.](https://trello.com/c/i0HgVrUV/145-constraint-de-performance-startdate-moet-binnen-de-festivals-start-en-eind-datum-vallen)

File: Database/Stored Procedures/Add_or_Update/add_or_update_performance.sql [hunk 1]
```diff
@@ -51,11 +51,10 @@ BEGIN
 
 		IF (@insert = 1)
 			BEGIN
-				INSERT INTO PERFORMANCE (artist_number, podium_schedule_number, festival_number, "start_date", start_time, play_time, min_prep_time) VALUES
+				INSERT INTO PERFORMANCE (artist_number, podium_schedule_number, festival_number, start_time, play_time, min_prep_time) VALUES
```
Comment (iYuriZ) [inline Database/Stored Procedures/Add_or_Update/add_or_update_performance.sql]: Start date staat er weer bij sinds PDM versie 7, dit graag terug zetten

File: Database/Stored Procedures/Add_or_Update/add_or_update_performance.sql [hunk 2]
```diff
@@ -51,11 +51,10 @@ BEGIN
...
 				(@artist_number,
 				 @podium_schedule_number,
 				 @festival_number,
-				 @start_date,
```
Comment (iYuriZ) [inline Database/Stored Procedures/Add_or_Update/add_or_update_performance.sql]: idem

File: Database/Stored Procedures/Add_or_Update/add_or_update_performance.sql [hunk 3]
```diff
@@ -78,7 +77,6 @@ BEGIN
 					artist_number = @artist_number,
 					podium_schedule_number = @podium_schedule_number,
 					festival_number = @festival_number,
-					"start_date" = @start_date,
```
Comment (iYuriZ) [inline Database/Stored Procedures/Add_or_Update/add_or_update_performance.sql]: idem

**Pull request merged**